cmd/snap-confine/snap-confine.apparmor.in

```diff
@@ -23,6 +23,8 @@
     /lib/@{multiarch}/libseccomp.so* mr,
 
     @LIBEXECDIR@/snap-confine mr,
+    # allow re-exec
+    /snap/{ubuntu-core,core}/*/@LIBEXECDIR@/snap-confine mr,
```

Comment:
I don't think we want this.

We want to run the *correct* snap-confine directly from snapd. Remember that snap run runs snap-confine. We can just run the right one all the time.